com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -146,6 +146,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added a built-in custom pass to draw object IDs.
 - Added an example in the documentation that shows how to use the accumulation API for high quality antialiasing (supersampling).
 - Added a SG node to get the main directional light direction.
+- Added 2 new functions to sample the custom buffer in custom passes (CustomPassSampleCustomColor and CustomPassLoadCustomColor), they avoid having to handle the RTHandleScale manually.
```

Comment:
Added new functions that sample the custom buffer in custom passes (CustomPassSampleCustomColor and CustomPassLoadCustomColor) to handle the RTHandleScale automatically.

Reply:
Fixed